.travis.yml

```diff
@@ -10,7 +10,8 @@ addons:
     packages:
     - postgresql-10
     - postgresql-client-10
-    - oracle-java8-set-default
+#Java 9 is used for elasticmq
+    - oracle-java9-set-default
```

Comment:
`- oracle-java9-set-default # Java 9 is used for elasticmq`

Reply:
Can you please add it like this way.

Reply:
Ok changed it like that way.